### ISSUE TITLE
featured image alt text in Blog Posts displays the title of the post

### ISSUE THREAD

**Issue by legaspielaine** [gambitph/Stackable#1214]
the image alt of the Blog post featured image should have the alt text of the image instead of the blog posts title:

<img width="297" alt="Screen Shot 2021-06-07 at 10 29 51 AM" src="https://user-images.githubusercontent.com/28699204/120951402-4c4dbd00-c77b-11eb-8ce6-5734c6435c59.png">
